flumine/streams/historicalstream.py

```diff
@@ -24,6 +24,8 @@ class FlumineMarketStream(MarketStream):
 
     def _process(self, data: list, publish_time: int) -> bool:
         for market_book in data:
+            if "id" not in market_book:
```

Comment:
Do we have to do this? This code is called a lot and heavily optimised, so want to remove any redundant function calls like this.